src/Phinx/Migration/AbstractMigration.php

```diff
@@ -177,6 +185,24 @@ public function getName()
         return get_class($this);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getEnvironment()
+    {
+        return $this->environment;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setEnvironment($environment)
```

Comment:
Can you remove this? I don't think changing the environment at runtime is a good idea

Reply:
Done!